virtcontainers/mount.go

```diff
@@ -277,6 +281,10 @@ func bindMount(ctx context.Context, source, destination string, readonly bool) e
 		return fmt.Errorf("Could not create destination mount point %v: %v", destination, err)
 	}
 
+	if dest, err := filepath.EvalSymlinks(destination); err == nil && !strings.HasPrefix(dest, filepath.Clean(topDir)) {
+		return fmt.Errorf("Invalid mount destination %s must not escape", destination)
+	}
+
```

Comment:
there is a "TOCTOU" here

Reply:
I dropped it as it also breaks vm template.